### PR TITLE
Import styles

### PR DESCRIPTION
**Pull request by NamrataRaikwar2002**
converted all html in react.

csb link : https://yb9p5s.csb.app/

### CODE REVIEW

**Review by BISINIGISHIVANI — APPROVED** [review]

File: src/pages/Authentication/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            <label for="login_input">Password</label>
+            <input
+              type="text"
+              className="login_input"
+              placeholder="Password"
+              required
+            />
+            <div className="forgot_password_div">
+              <input type="checkbox" name="1" className="rememberme" required />
+              <label for="rememberme">Remember me</label>
```
Comment (Sudhanshusoni45) [inline src/pages/Authentication/Login.jsx]: Please change for to htmlFor in all the labels

File: src/pages/Authentication/Signup.jsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        <form>
+          <div className="login_div">
+            <h2 className="createAccount">Signup</h2>
+            <label for="login_input">First Name</label>
+            <input
+              type="text"
+              className="login_input"
+              placeholder="Enter you first name"
+              required
+            />
```
Comment (Sudhanshusoni45) [inline src/pages/Authentication/Signup.jsx]: Whenever there is label and input together you can give id to input and htmlFor to labels their values being same, this helps in accessibility.

File: src/pages/Home/component/LandingFeature.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const LandingFeature = ({ featureCardImg, featureCardName }) => {
+  return (
+    <div className="home_features">
+      <img className="featureImg" src={featureCardImg} alt="image" />
```
Comment (Sudhanshusoni45) [inline src/pages/Home/component/LandingFeature.jsx:4]: Try to give more descriptive values to alt

Comment (NamrataRaikwar2002) [reply src/pages/Home/component/LandingFeature.jsx:4]: thank you sudhanshu i will keep it in mind.

**Review by Sudhanshusoni45 — APPROVED** [review]

**Pull request merged**